### ISSUE TITLE
Great Work When Can We Expect An Update !!!

### ISSUE THREAD

**Issue by GatekeeperZA** [Stratehm/stratum-proxy#81]
Thank you very much for providing the package. 
Please can you advise when there could / might be a release update.

Thanks.

**Comment by chrysophylax69** [comment]
stratehm has already published that this project is pretty much abandoned / no longer supporting it ...

he is a great dev - but has too many commitments currently in 'real life' ...

we would like to find a java dev that could maintain this package - so we could take it on ( cwi-projects ) - but devs for opensource ( ie - no pay ) are very hard to come by these days ... though we would do a nice deal with them ... hint hint stratehm ;) ...

#crysx

**Comment by GatekeeperZA** [comment]
Would be great for him to get this going. catching your hint.

**Comment by barrystyle** [comment]
@GatekeeperZA I don't think you should hassle dev/team unless you've contributed BTC etc; is just rude otherwise. chrysophylax69 has hinted that progress will occur if deal is struck; might pay to work something out in monetary terms.

Otherwise, use software as it stands, you get what you pay for.

**Comment by albsoft1** [comment]
I fully agree with @barrystyle and would like to add further that if a "symbolic" contribution from my end would let @GatekeeperZA and/or any other dev/team correct some small thinks on current stratehm version, am willing to further discuss this.

**Comment by GatekeeperZA** [comment]
No appologies you've got it wrong I'm all in for the deal have no issues
here is someone can come to an agreement to improve stratum would be
excellent.

**Comment by mooleshacat** [comment]
5 years later ... nothing.

Not sure why I thought my tickets would get answered after seeing this lol

**Comment by barrystyle** [comment]
i think just be grateful this bloke even put his stratum proxy code up in the first place; let alone compile it up for you muppets.. ;)
